scripts/gen_plots.py

```diff
@@ -0,0 +1,47 @@
+"""Generates plots using icubam.analytic"""
+import logging
+import sys
+from pathlib import Path
+
+from absl import app, flags
+
+from icubam import config
+from icubam.db.store import create_store_factory_for_sqlite_db
+from icubam.analytics import data, plots
+
+flags.DEFINE_string('config', config.DEFAULT_CONFIG_PATH, 'Config file.')
+flags.DEFINE_string('output_dir', '/tmp', 'Output dir for plots.')
+flags.DEFINE_string('plot_name', 'base_dashboard_plots', 'Plot name.')
+FLAGS = flags.FLAGS
```

Comment:
We kind of have already this script in `analytics.__main__` done with click which has a significantly better documentation.

Reply:
Yeah, but since we are using dedicated .py files for all other scripts it seemed a bit non-standard to use the __main__ trick only here.  Happy to reconsider.

Reply:
If they are part of the main module and use `click` library, we could have auto-generated the docs for them see e.g. https://paris-saclay-cds.github.io/ramp-docs/ramp-board/stable/cli.html

Otherwise in the current situation the only way for an external contributor to discover what they do is to run `script/<name>.py --help` one by one or read the source code.

Reply:
Agreed, but I'm not sure that's worth the inconsistency relative to our other scripts.  @olivierteboul do you have an opinion?

Reply:
I would keep it like this for consistency with other ways to run scripts.